routes/index.tsx

```diff
@@ -54,25 +62,12 @@ export const handler: Handlers<HomePageData, State> = {
   },
 };
 
-function PageSelector(props: { currentPage: number; lastPage: number }) {
+function TimeSelector() {
   return (
-    <div class="flex justify-center py-4 mx-auto">
-      <form class="inline-flex items-center gap-x-2">
-        <input
-          id="current_page"
-          class={`bg-transparent rounded rounded-lg outline-none w-full border-1 border-gray-500 hover:border-black transition duration-300 disabled:(opacity-50 cursor-not-allowed) rounded-md px-2 py-1 dark:(hover:border-white)`}
-          type="number"
-          name="page"
-          min="1"
-          max={props.lastPage}
-          value={props.currentPage}
-          // @ts-ignore: this is valid HTML
-          onchange="this.form.submit()"
-        />
-        <label for="current_page" class="whitespace-nowrap align-middle">
-          of {props.lastPage}
-        </label>
-      </form>
+    <div class="flex justify-center">
```

Comment:
Please add a note above saying these links do not preserve current URL queries. E.g. if `?page=2`, that'll be removed once one of these links is clicked. This behaviour is fine in this particular context but might need to change in the future as more URL queries control the item list view.